internal/domain/indexer.go

```diff
@@ -250,6 +250,8 @@ type IndexerIRCParseLine struct {
 type IndexerIRCParseMatch struct {
 	TorrentURL  string   `json:"torrenturl"`
 	TorrentName string   `json:"torrentname"`
+	TorrentHash string   `json:"torrenthash"`
```

Comment:
This can be removed since it's no longer used

Reply:
Done in `5e95dc6`.